php/cache/class-cache-point.php

```diff
@@ -476,6 +476,48 @@ protected function get_parent_cache_point( $url ) {
 		return $parent;
 	}
 
+	/**
+	 * Get all cache items for a cache point.
+	 *
+	 * @param string|int $cache_point_id_url The cache point ID or URL.
+	 * @param bool       $id_only            Flag to get ID's only.
+	 *
+	 * @return \WP_Post[]
+	 */
+	public function get_cache_items( $cache_point_id_url, $id_only = false ) {
+		$items = array();
+		if ( ! is_int( $cache_point_id_url ) ) {
+			$cache_point = $this->get_cache_point( $cache_point_id_url );
+		} else {
+			$cache_point = get_post( $cache_point_id_url );
+		}
+		if ( ! is_null( $cache_point ) ) {
+
+			$params = array(
+				'post_type'              => self::POST_TYPE_SLUG,
+				'posts_per_page'         => 1,
```

Comment:
@DavidCramer 

Any reason why we don't do in batches of 100, for instance?

Reply:
Typo! fixing.